integration_test/cases/repo.exs

```diff
@@ -632,6 +633,7 @@ defmodule Ecto.Integration.RepoTest do
     assert TestRepo.aggregate(query, :count, :visits) == 3
   end
 
+  @tag :insert_cell_wise_defaults
```

Comment:
Sorry but I don' understand exactly what this tag is about. Can you please explain why this does not work on sqlite_ecto2?